app/collins/models/LshwHelper.scala

```diff
@@ -86,6 +92,41 @@ object LshwHelper extends CommonHelper[LshwRepresentation] {
     )
   }
 
+  protected def reconstructGpu(meta: Map[Int, Seq[MetaWrapper]]): FilteredSeq[Gpu] = {
```

Comment:
ideally this would be made more legible to make support burden easier going forward